math_functions/test/math_functions_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule MathFunctionsTest do
+  use ExUnit.Case
+  doctest MathFunctions
+
```

Comment:
This might be useful: https://hexdocs.pm/ex_unit/ExUnit.Assertions.html#assert_in_delta/4